.github/workflows/release_linux_pytorch_wheels.yml

```diff
@@ -15,11 +38,10 @@ on:
         description: CloudFront URL pointing to Python index
         type: string
         default: "d25kgig7rdsyks.cloudfront.net/v2"
-      # rocm_version:
-      #   description: ROCm version to install (e.g. "==1.0", ">1.0")
-      #   type: string
-  schedule:
-    - cron: "0 2 * * *"  # Nightly at 2 AM UTC
+      rocm_version:
+        description: ROCm version to pip install
+        type: string
```

Comment:
What happens if this is omitted with workflow_dispatch? Does it use the latest?

Reply:
For a nightly build pip will fetch the latest but I haven't checked how pip will behave if you point to the dev bucket (as this has some dev releases but also rc versions).